ding/framework/middleware/functional/data_fetcher.py

```diff
@@ -10,9 +11,10 @@
 
 def offpolicy_data_fetcher(cfg: EasyDict, buffer_: Buffer) -> Callable:
 
-    def _fetch(ctx: "Context"):
+    def _push_and_fetch(ctx: "Context"):
+        for t in ctx.trajectories:
```

Comment:
Put this property on ctx, use OnlineRLContext please

Reply:
added it